recipes/ciocheck/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ciocheck" %}
+{% set version = "0.1.1" %}
+{% set hash_type = "sha1" %}
+{% set hash = "8cfedcc1d462f6f55ab33d2ea08b91ad55dca6e9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_rev: v0.1.1
+  git_url: https://github.com/ContinuumIO/ciocheck.git
```

Comment:
That is the policy. 😄 Unless of course there is a good reason not to use some form of tarball.